install.js

```diff
@@ -73,50 +69,49 @@ function requstBinary(downloadURL, baseName) {
     const filePath = path.join(process.cwd(), binaryName);
     const outFile = fs.openSync(filePath, 'w');
 
-    const requestOptions = {
-      uri: downloadURL,
-      method: 'GET',
-    };
-    const client = request(requestOptions);
```

Comment:
the old `request` library was still being used here, replaced this with `axios` as well